app/soundcloud.js

```diff
@@ -50,8 +50,9 @@ module.exports = class SoundCloud extends Events {
 
   isPlaying() {
     return new Promise(resolve => {
-      ipcMain.once('isPlaying', (_, isPlaying) => {
-        resolve(isPlaying)
+      ipcMain.once('isPlaying', (_, isPlaying, icon) => {
+        var values = [isPlaying, icon]
+        resolve(values)
```

Comment:
I agree the values might grow. In order to add the scrubbing feature to the touchbar I will need access to the track length and current playing position. I really don't know too much about JavaScript so I appreciate your help!